plugins/cachekey/pattern.cc

```diff
@@ -452,6 +452,18 @@ MultiPattern::name() const
   return _name;
 }
 
+bool
+MultiPattern::capture(const String subject, StringVector &result) const
+{
+  bool res = false;
+  for (auto p : this->_list) {
+    if (nullptr != p && p->process(subject, result)) {
```

Comment:
can you comment why this is calling `Pattern::process()` instead of `Pattern::capture()`. Its unclear to me.

Reply:
`Pattern::process()` supports `capture` or `capture-and-replace` depending on whether a replacement string is specified. 

It seemed to me that there might be much more different ways to process a `MultiPattern` than a `Pattern` so I thought I would call the corresponding method `capture()` instead of `process()` which would be closer to what it does but this seems to be confusing people in a different way.

I will change the `MultiPattern::capture()` to `MultiPattern::process()`